<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dataCenter</class>
 <widget class="QWidget" name="dataCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="createDatabase">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>创建数据库</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createTable">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>创建数据表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="queryData">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>210</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查询数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="insertData">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>插入数据</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tableEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="insertEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>671</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="queryEdit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="queryShow">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>256</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="toastLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QListWidget" name="showAllDatabase">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>包含表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>310</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>包含列(字段)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>操作说明：
1.选中相应数据库后在创建表框输入以创建表;
2.点击相应表查看字段并进行插入/修改操作;
3.选中表(数据库)并输入语句进行查询操作;
4.谨慎删除操作!</string>
   </property>
  </widget>
  <widget class="QListWidget" name="showAllTables">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>当前数据库</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>341</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
